envpool/entry.py

```diff
@@ -35,3 +35,25 @@
     task=game,
     base_path=base_path,
   )
+
+# Classic Control
+
+register(
+  task_id="CartPole-v0",
+  import_path="envpool.classic_control",
+  spec_cls="CartPoleEnvSpec",
+  dm_cls="CartPoleDMEnvPool",
+  gym_cls="CartPoleGymEnvPool",
+  max_episode_steps=200,
+  reward_threshold=195.0,
+)
+
+register(
+  task_id="CartPole-v1",
+  import_path="envpool.classic_control",
+  spec_cls="CartPoleEnvSpec",
+  dm_cls="CartPoleDMEnvPool",
+  gym_cls="CartPoleGymEnvPool",
+  max_episode_steps=500,
+  reward_threshold=475.0,
+)
```

Comment:
I think putting register code in the per env `__init__` is better? People can then register their custom envs to the locally installed envpool without having merge into upstream.

Reply:
We can open another issue to track this.

Reply:
but what about circular dependency?

Reply:
https://github.com/sail-sg/envpool/pull/26 resolves this.